rflx/expression.py

```diff
@@ -1710,8 +1710,21 @@ def _simplified(self, relation_operator: Callable[[Expr, Expr], bool]) -> Expr:
         }
         if (relation_operator, left, right) in mapping:
             return mapping[(relation_operator, left, right)]
-        if isinstance(left, Number) and isinstance(right, Number):
-            return TRUE if relation_operator(left, right) else FALSE
+        if isinstance(left, (Number, Aggregate)) and isinstance(right, (Number, Aggregate)):
+            if isinstance(left, Number) and isinstance(right, Number):
+                return TRUE if relation_operator(left, right) else FALSE
+            if isinstance(left, Aggregate) and isinstance(right, Aggregate):
+                return TRUE if left.elements == right.elements else FALSE
```

Comment:
It should be possible to handle aggregates in the same way as numbers when implementing `__eq__` for `Aggregate`.